test/test_functional_tensor.py

```diff
@@ -531,8 +532,8 @@ def test_resize(device, dt, size, max_size, interpolation):
         tensor = tensor.to(dt)
         batch_tensors = batch_tensors.to(dt)
 
-    resized_tensor = F.resize(tensor, size=size, interpolation=interpolation, max_size=max_size)
-    resized_pil_img = F.resize(pil_img, size=size, interpolation=interpolation, max_size=max_size)
+    resized_tensor = F.resize(tensor, size=size, interpolation=interpolation, max_size=max_size, antialias=True)
```

Comment:
I don't think v2 tests need changes at all:

1. The tensor kernel, e.g. `resize_image_tensor` only branches on `antialias` to bail out in case it is set together with an `interpolation` that doesn't support it: 

   https://github.com/pytorch/vision/blob/2cd25c1a05012a3720a87f20cec436811fadeedd/torchvision/prototype/transforms/functional/_geometry.py#L150-L153

   Thus, our `sample_inputs_fn`'s, that should cover all _valid_ branches, don't set it at all:

   - https://github.com/pytorch/vision/blob/2cd25c1a05012a3720a87f20cec436811fadeedd/test/prototype_transforms_kernel_infos.py#L265
   - https://github.com/pytorch/vision/blob/2cd25c1a05012a3720a87f20cec436811fadeedd/test/prototype_transforms_kernel_infos.py#L275
   - https://github.com/pytorch/vision/blob/2cd25c1a05012a3720a87f20cec436811fadeedd/test/prototype_transforms_kernel_infos.py#L277
2. Our reference test check the image tensor against the corresponding PIL kernel, e.g. `resize_image_pil`. Since PIL "hardcodes" `antialias=True`, we need to do the same:

   https://github.com/pytorch/vision/blob/2cd25c1a05012a3720a87f20cec436811fadeedd/test/prototype_transforms_kernel_infos.py#L280-L286

   https://github.com/pytorch/vision/blob/2cd25c1a05012a3720a87f20cec436811fadeedd/test/prototype_transforms_kernel_infos.py#L304-L309

   (I saw that I messed up the `antialias` value here. Will send a PR soon)
3. Since the dispatcher, e.g. `resize`, no longer contains any logic besides the dispatching, we just use the kernel inputs and check if they are dispatched to the correct kernel:

   https://github.com/pytorch/vision/blob/2cd25c1a05012a3720a87f20cec436811fadeedd/test/prototype_transforms_dispatcher_infos.py#L158-L170

   - https://github.com/pytorch/vision/blob/2cd25c1a05012a3720a87f20cec436811fadeedd/test/test_prototype_transforms_functional.py#L358
   - https://github.com/pytorch/vision/blob/2cd25c1a05012a3720a87f20cec436811fadeedd/test/test_prototype_transforms_functional.py#L383
   - https://github.com/pytorch/vision/blob/2cd25c1a05012a3720a87f20cec436811fadeedd/test/test_prototype_transforms_functional.py#L418

Reply:
We're still getting hundreds of warnings just in `test_prototype_transforms.py`. Do you know how to cleanly get rid of those?

```
test/test_prototype_transforms.py: 456 warnings
  /home/nicolashug/dev/vision/torchvision/transforms/functional.py:1621: UserWarning: The default value of the antialias parameter will change from None to True in v0.17, in order to be consistent across the PIL and Tensor backends. To suppress this warning, directly pass antialias=True (recommended, future default), antialias=None (current default, which means False for Tensors and True for PIL), or antialias=False (only works on Tensors - PIL will still use antialiasing).
```

Reply:
Fixed in f6c8140cad81303747dcd5b1ff3b61e79af1ea4a. It was just two transforms that needed fixing. They generated ~400 warnings since we tested a lot of parametrizations on them.